moralization/spacy_model.py

```diff
@@ -0,0 +1,229 @@
+from logging import exception
+import spacy
+from spacy.tokens import DocBin
+from spacy.matcher import Matcher
+
+from moralization import InputOutput
+import glob
+import os
+import random
+from collections import defaultdict
+import pathlib
+from sklearn.metrics import classification_report
+
+
+class Machine_Learning:  # name is subject to change
```

Comment:
The name of the modules and classes need to be generalized - for example, `machine_learning.py` and `spacy_model.py` don't exactly contain what I would expect (maybe the former is not so bad, but spacy_model can be misleading).